tests/suites/test_suite_pk.function

```diff
@@ -1861,9 +1867,14 @@ void pk_psa_wrap_sign_ext(int pk_type, int key_bits, int key_pk_type, int md_alg
     }
 
     /* Turn PK context into an opaque one. */
-    TEST_EQUAL(mbedtls_pk_wrap_as_opaque(&pk, &key_id, psa_alg,
-                                         PSA_KEY_USAGE_SIGN_HASH,
-                                         PSA_ALG_NONE), 0);
+    TEST_EQUAL(mbedtls_pk_get_psa_attributes(&pk, PSA_KEY_USAGE_SIGN_HASH, &key_attr), 0);
+    /* Tweak the algorithm associated with the PSA key because get_psa_attributes() returns
+     * a PSA_ALG_RSA_PSS_ANY_SALT(), but mbedtls_pk_sign_ext() requires a PSA_ALG_RSA_PSS().*/
```

Comment:
If we have to do this, so will some users. Can't we make `pk_sign_ext()` work with `PSS_ANY_SALT()`? Ideally we want users to be able to just do `get_psa_attributes()` + `import_into_psa()` without tuning anything in as many cases  as possible.